src/pudl/extract/eia860.py

```diff
@@ -69,29 +69,29 @@ def get_dtypes(page, **partition):
 
 # TODO (bendnorman): Add this information to the metadata
 raw_table_names = (
-    "raw_boiler_cooling_eia860",
-    "raw_boiler_generator_assn_eia860",
-    "raw_boiler_info_eia860",
-    "raw_boiler_mercury_eia860",
-    "raw_boiler_nox_eia860",
-    "raw_boiler_particulate_eia860",
-    "raw_boiler_so2_eia860",
-    "raw_boiler_stack_flue_eia860",
-    "raw_cooling_equipment_eia860",
-    "raw_emission_control_strategies_eia860",
-    "raw_emissions_control_equipment_eia860",
-    "raw_fgd_equipment_eia860",
-    "raw_fgp_equipment_eia860",
-    "raw_generator_eia860",
-    "raw_generator_existing_eia860",
-    "raw_generator_proposed_eia860",
-    "raw_generator_retired_eia860",
-    "raw_multifuel_existing_eia860",
-    "raw_multifuel_retired_eia860",
-    "raw_ownership_eia860",
-    "raw_plant_eia860",
-    "raw_stack_flue_equipment_eia860",
-    "raw_utility_eia860",
+    "raw_eia860__boiler_cooling",
```

Comment:
Looking at a whole stack of these, I'm shocked at how much nicer it is to have the double underscore as a sort of "column separator!"